### PR TITLE
Added presentation outline for Puzzle Stealth Networking Team

### PR DESCRIPTION
**Pull request by patrickrhee**

### CODE REVIEW

File: adv_topics/networking.md [hunk 1]
```diff
@@ -4,7 +4,44 @@
 * 
 
 ## Presentation II (Oct. 28):
-* 
+* Very Briefly Refresh/Review Presentation I
+	* Layers of networking
+		* What is each layer responsible for?
```
Comment (nfarnan) [inline adv_topics/networking.md:9]: Don't need to review this

File: adv_topics/networking.md [hunk 2]
```diff
@@ -4,7 +4,44 @@
...
+
+* UDP and TCP
```
Comment (nfarnan) [inline adv_topics/networking.md:11]: This is fine as the purposes of TCP/UDP were under covered in the first presentation.

File: adv_topics/networking.md [hunk 3]
```diff
@@ -4,7 +4,44 @@
...
+			* Packet discarding
+
+* What will we be using
+	* SDL_net
+		* What is it?
+		* Example code with functions
+			* What is each function doing?
+			* Use in our program
+	* Client-server network
+		* Server implementation
```
Comment (nfarnan) [inline adv_topics/networking.md:38]: Be careful with a code overview, this can be tricky to do effectively in slides.

**Review by nfarnan — CHANGES_REQUESTED** [review]

File: adv_topics/networking.md (hunk 3 above)
Comment (patrickrhee) [reply adv_topics/networking.md:34]: Got it. I was thinking of having a simplified list of steps our program executes to make networking happen, but would it still be too specific/advanced for the audience?
(e.g: "Step 1 - We use our IP/Port# to create a listening socket")

Comment (nfarnan) [reply adv_topics/networking.md:34]: No, I think that should be fine

**Review by nfarnan — APPROVED** [review]

**Pull request merged**